Say/ViewController.swift

```diff
@@ -90,7 +93,29 @@ class ViewController: NSViewController {
             }
         }
     }
-
+    
+    @IBAction func selectText(_ sender: NSTextField){
+        let URLString = sender.stringValue
+        let url = URL(string: sender.stringValue)!
+        var data: NSData? = nil
+        data=NSData.init(contentsOf: url)//NSurlData
```

Comment:
It's better to be changed like this. It's about code style. 😄
`data = NSData.init(contentsOf: url)`